### DISCUSSION TITLE
setup discussion platform

### DISCUSSION THREAD

**Discussion by geroldmeisinger** [ivanperez-keera/dunai#329] [General]
I'd really like a discussion platform for Dunai and Yampa which is open and accessible.

The old mailing list is no longer visible (and was archaic). Some platform are proprietary and closed, with no access to the raw data (Discord, Reddit). Classic forums and web boards require hosting, setup and maintenance (phpBB). 

A recurring scheme I've seen with some smaller projects is to setup an additional repository "project-discuss" where issues are misused for open discussion without spamming the original issues. Anyone cloning the repo is likely already registered on github.

Another way would be to make the `yampa` tag on [Stackoverflow](https://stackoverflow.com/questions/tagged/yampa) more visible in the readme. With the potential downside that some discussions could get lost due to missing or wrong tags ("should I use dunai bearriver or yampa or haskell+FRP?").

What are your opinions?

**Comment by ivanperez-keera** [comment]
The old mailing list does exist though. But we do need to move out.

Other alternatives apart from what you mentioned:
- Haskell-cafe
- Discussion tabs on github (can be enabled).
- Group on google groups

I'm not fond of and very unlikely to participate in:
- IRC
- Reddit

**Comment by ivanperez-keera** [comment]
I'm also unlikely to participate in anything that produces sounds or has a similar interface or interaction model to IRC (like slack), or that requires constant monitoring to participate.

**Comment by geroldmeisinger** [comment]
I'm unlikely to participate on Reddit, IRC, Slack etc. either for the same reasons.

> Discussion tabs on github (can be enabled).

Oh, didn't know that. Well, that sounds like the way forward?

**Comment by ivanperez-keera** [comment]
We can try it. We just need to be aware that if we ever move out of github, those may not port over. (Although, if I'm not mistaken, they use the same implementation as the issues.)

**Reply by geroldmeisinger** [reply]
Quick research:
[Download all your data](https://github.blog/2018-12-19-download-your-data/) (may or may not include discussions)
You can access [issues via API](https://api.github.com/repos/ivanperez-keera/dunai/issues)
I haven't found any [API call for discussions](https://api.github.com/repos/ivanperez-keera/dunai/discussions) (not working)
however there is one for [Team discussiosn](https://docs.github.com/en/rest/teams/discussions)
github issue: [Possibility to export discussions ? #3315](https://github.com/community/community/discussions/3315)

**Comment by ivanperez-keera** [comment]
There is it. Now we have a discussion platform.
